src/routes/products.js

```diff
@@ -0,0 +1,142 @@
+import express from "express" 
+import { PrismaClient } from "@prisma/client";
+import asyncHandler from "../async-handler.js";
+ 
+
+
+const prisma = new PrismaClient();
+const ProductsRoute = express.Router();     
+  
+ProductsRoute
+  .route('/')
+  .post(asyncHandler(async(req,res)=>{
+    const {name,description,price,tags} = req.body
+    const Newdata = await prisma.product.create({
```

Comment:
변수명에 조금더 명확성을 주기 위해 저는 아래와 같이 변수명을 지었을 것 같습니다.
```suggestion
    const products = await prisma.product.create({
```
or

```suggestion
    const createdProduct = await prisma.product.create({
```